1017/en.md

```diff
@@ -0,0 +1,118 @@
+# LOJ 1017 - Brush(III)
+
+### Tags:
+DP, Cartesian co-ordinate System, Binary Search
+___
+### Problem:
+Given the co-ordinate of dust point. We have to find how many dust can we clean with w width brush after exactly K Moves.
+___
+### Helpful Resources:
+ -  1.[Wiki: Cartesian Co-ordinate](https://en.wikipedia.org/wiki/Cartesian_coordinate_system)
+ -  2.[Geekforgeeks: DP](https://www.geeksforgeeks.org/dynamic-programming/)
+ -  3.[Youtube: BACS(DP)](https://www.youtube.com/watch?v=cbgdSX2pXcQ&t=2479s&ab_channel=BangladeshAdvancedComputingSociety-BACS)
+ -  4.[Geekforgeeks : Binary Search](https://www.geeksforgeeks.org/binary-search/)
+___
+### Solution:
+Basically, we need only y axis point.  
+<img width="1200" alt="initial" src="https://github.com/ahrifatarafat/Lightoj-Tutorial/blob/main/LOJ_1017/LOJ_1017.JPG">
+Here we see (0,0); (0,5); (20,5); (100,8); the four points then which way we solve we don’t need x axis points.
+Then sort the y axis points and then we calculate in every moves how many dusts we can clean.
+And also observe that if we start from non-overlapping points then we can clean most of them.
+We first stored if we clean this dust then how many of them doesn’t overlapped before we can clean. And also how many of them we can take from front for clean. Then We can check in every moves how many dust we can clean.
+We run a for loop from 0 to k for every moves and iterate over y axis point and calculate the maximum number of dust we can clean on index j of y axis after i moves and stored in dp[j].

```

Comment:
Ambiguous. Perhaps due to grammatical problems?
Also, pleases clearly state what your dp function does, states, transitions and base cases.